core/src/dird/ua_status.cc

```diff
@@ -478,6 +478,24 @@ static bool show_scheduled_preview(UaContext*,
   return true;
 }
 
+std::string get_subscription_status_signature_source_text(UaContext* ua,
+                                                          const char* timestamp)
+{
+  std::string salt("SECRETSALT");
+  PoolMem subscriptions(PM_MESSAGE);
+  PoolMem query(PM_MESSAGE);
+  OutputFormatter output_text
+      = OutputFormatter(pm_append, (void*)&subscriptions, nullptr, nullptr);
```

Comment:
casts to `void*` are implicit and c-style casts are evil.
```suggestion
      = OutputFormatter(pm_append, &subscriptions, nullptr, nullptr);
```

Reply:
done.